### ISSUE TITLE
The branch parameter is not working

### ISSUE THREAD

**Issue by JasonXu-YK** [joalmeid/pipeline-triggerer#2]
I pass branch parameter but the trigger pipeline always use the master branch

**Comment by igor-a-yastrebov** [comment]
same on my end. Spent a day building a solution around this plugin, and very last step does not work :((((

**Comment by nagarajujalapala-sage** [comment]
Yes the branch parameter is not working. It is using master branch either ways

**Comment by loixiyo** [comment]
Hi, 

any update regarding this issue?

**Comment by PrudhviMadasu** [comment]
Same here
